modules/serverless-logic-global-config-customization.adoc

```diff
@@ -0,0 +1,130 @@
+|{ServerlessLogicProductName} base builder image used in the internal Dockerfile to build workflow applications in preview profile. If empty, the {ServerlessLogicOperatorName} uses the default Apache community image based on the current {ServerlessLogicOperatorName} version.
+
+|`sonataFlowDevModeImageTag`
+|`registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:{ServerlessProductVersion}`
+|The image to use to deploy {ServerlessLogicProductName} workflow images in devmode profile. If empty, the {ServerlessLogicOperatorName} uses the default Apache community image based on the current {ServerlessLogicOperatorName} version.
+
+|`builderConfigMapName`
+|`logic-operator-rhel8-builder-config`
+|The default name of the builder config map in the {ServerlessLogicOperatorName} namespace.
+
+|`postgreSQLPersistenceExtensions`
+|next column
+|Quarkus extensions required for workflows persistence. These extensions are used by the {ServerlessLogicOperatorName} builder in cases where the workflow being built has configured PostgreSQL persistence.
+
+|`kogitoEventsGrouping`
+|`true`
+|When set to `true`, configures every workflow deployment with the `gitops` or `preview` profiles to send accumulated workflow status change events to the Data Index service, reducing the number of produced events. You can set the value to `false` to send individual events.
+
+|`kogitoEventsGroupingBinary`
+|`true`
+|When set to `true`, the accumulated workflow status change events are sent in binary mode, reducing the size of the produced events. You can set the value to `false` to send plain JSON events.
+
+|`kogitoEventsGroupingCompress`
+|`false`
+|When set to `true`, the accumulated workflow status change events, if sent in binary mode, are zipped at the cost of some performance.
+
+|===
+
+You can edit this by updating the `logic-operator-controllers-config` config map by using the `oc` command-line tool.
+
+[id="serverless-logic-global-configuration-changes-impact_{context}"]
+== Impact of global configuration changes
+
+When you update the global configurations, the changes immediately affect only newly created resources. For example, if you change the `sonataFlowDevModeImageTag` property and already have a workflow deployed in dev mode, the {ServerlessLogicOperatorName} does not roll out a new deployment with the updated image configuration. Only new deployments reflect the changes.
+
+[id="serverless-logic-customizing-base-builder-image_{context}"]
+== Customizing the base builder image
+
+You can directly change the base builder image in the Dockerfile used by the {ServerlessLogicOperatorName}.
+
+Additionally, you can specify the base builder image in the `SonataFlowPlatform` configuration within the current namespace. This ensures that the specified base image is used exclusively in the given namespace.
+
+.Example of `SonataFlowPlatform` with a custom base builder image:
```

Comment:
```suggestion
.Example of `SonataFlowPlatform` with a custom base builder image
```

Sorry I might have missed the `:` removal in the first review.

Reply:
Np! But thank you for catching this out :)